eve/main.yml

```diff
@@ -753,7 +753,9 @@ stages:
           name: Trigger upgrade and downgrade test stages simultaneously
           stage_names:
             - single-node-upgrade-centos
-            - single-node-downgrade-centos
+            # NOTE: Deactivate minor downgrade on this branch
+            # See: https://github.com/scality/metalk8s/issues/1750
+            #- single-node-downgrade-centos
```

Comment:
We upgrade from 3.4.3 to 3.4.13, shouldn't downgrade work for patch versions? (I tried looking around their docs, but there doesn't seem to be mentions of downgrade)

Reply:
Hmm right, worth a try, I will merge this PR as is for the moment and I will check this :+1:

Reply:
Btw I also forgot to disable the downgrade in salt in the defaults.yaml so anyway I will do another PR for downgrade (either to really disable it or to enable it back :smile: )

Reply:
Patch version etcd downgrade is supported (at least from 3.4.13 to 3.4.3) so this commit get reverted in #3023